department-of-veterans-affairs/abd-vro
# Adjustments to flyway scripts based on Dimitris suggestions

MCP-1496

## Description



### What was the problem?

Issues with column names, data types, keys, and general best practice

### How does this fix it?

Changes made to column names and data types through new flyway script

### Jira Tickets



- [MCP-1496](https://amida.atlassian.net/browse/MCP-1496)

## How to test this PR

- Step 1 Pull into local and complete a build
- Step 2 Check the db for updated tables

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
